Firestore/core/test/firebase/firestore/immutable/testing.h

```diff
@@ -144,6 +144,15 @@ Container ToMap(const std::vector<int>& values) {
   return result;
 }
 
```

Comment:
Unrelated to this PR: this file contains some helper functions that apply an algorithm to a container, e.g. `Sorted`, `Reversed`. Skimming through Abseil code, I noticed they have `algorithm/container.h`, which has wrappers over all applicable standard library algorithms that take a container instead of a pair of iterators (e.g. [c_sort](https://github.com/abseil/abseil-cpp/blob/master/absl/algorithm/container.h#L829)). It's not an exact copy (because `Sorted` also returns a new vector), but perhaps it could still be useful for our code.

Reply:
These seem like a pretty marginal improvement:
```c++
std::sort(foo.begin(), foo.end());
```
vs
```
absl::c_sort(foo);
```
I'll grant that it's shorter, but the original isn't horrible. If you're passionate about it, please feel free :-).

Reply:
Nope, just wanted to share in case you find it useful elsewhere.

One day, we'll be able to use range-v3 for this.